internal/cmdfndoc/cmdfndoc.go

```diff
@@ -69,8 +69,7 @@ func (r *Runner) runE(c *cobra.Command, _ []string) error {
 	var out, errout bytes.Buffer
 	dockerRunArgs := []string{
 		"run",
-		"--rm",                         // delete the container afterward
-		"-a", "STDOUT", "-a", "STDERR", // attach stdin, stdout, stderr
```

Comment:
This is not necessary and has been dropped in https://github.com/GoogleContainerTools/kpt/pull/3362#pullrequestreview-1036496419